pkg/sdk/views_def.go

```diff
@@ -201,7 +201,11 @@ var ViewsDef = g.NewInterface(
 			OptionalStartsWith().
 			OptionalLimit(),
 	).
-	ShowByIdOperation().
+	ShowByIdOperation(
```

Comment:
I feel like an additional layer of abstraction would be better. I think receiving an enum for common presets would be sufficient (like in the case of DescribeOperation), e.g.
```go
// enum names could be better, but that's what I imagined it like
ShowByIdOperation(g.ShowByIdLikeFiltering)
ShowByIdOperation(g.ShowByIdInDatabaseWithLikeFiltering)
ShowByIdOperation(g.ShowByIdInSchemaWithLikeFiltering)
```

Reply:
This is exactly what I wanted to point out. I would even say that it would be possible to base the `In` choice on the identifier type (because for schema objects with search in schema, for db objects we search in db, etc.). But the one suggested by @sfc-gh-jcieslak should be enough.